targetcli/ui_backstore.py

```diff
@@ -651,6 +651,12 @@ def ui_command_create(self, name, size, cfgstring, wwn=None,
                             % (name, size))
         return self.new_node(ui_so)
 
+    def ui_command_mediachange(self, name, size, cfgstring):
+        size = human_to_bytes(size)
+        config = self.handler + "/" + cfgstring
+
+        ok, errmsg = self.iface.MediaChange('(sts)', name, size, config)
+        if ok != 0: raise ExecutionError("MediaChange failed")
```

Comment:
Why is the command not on the device, so you don't have to pass in the device name?

Do we also want to limit this command to certain types of devices like optical? Maybe that check should go in tcmu-runner though. Or, should the command be more generic like just "reconfigure" and then let tcmu-runner translate that and do the right thing.